cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMongoBaseSink.java

```diff
@@ -98,33 +107,35 @@ protected void setBackend(MongoBackendImpl backend) {
     protected MongoBackendImpl getBackend() {
         return backend;
     } // getBackend
-    
+
     @Override
     public void configure(Context context) {
         super.configure(context);
-        
+
         mongoHosts = context.getString("mongo_hosts", "localhost:27017");
         LOGGER.debug("[" + this.getName() + "] Reading configuration (mongo_hosts=" + mongoHosts + ")");
         mongoUsername = context.getString("mongo_username", "");
         LOGGER.debug("[" + this.getName() + "] Reading configuration (mongo_username=" + mongoUsername + ")");
         // FIXME: mongoPassword should be read as a SHA1 and decoded here
         mongoPassword = context.getString("mongo_password", "");
         LOGGER.debug("[" + this.getName() + "] Reading configuration (mongo_password=" + mongoPassword + ")");
-        
+        mongoAuthSource = context.getString("mongo_auth_source", "");
```

Comment:
I thing it's more usuall that the default authSource is "admin". It would be better if the default value of mongo_auth_source is "admin".

Reply:
IMHO authSource is necessary to be configured and not fixed as default constant value to admin for security reason. Admin is a sensible and critical database with rights and high privileges. Giving the possibility to configure the authentication database will allow to provide other specific databases with some restrictions and with just the necesary privileges with a set of user depending of the uses and applications needed and it is more secure tha using only the admin database.

Reply:
You are right, I misunderstood the "defaultauthdb" value, it actually is the db where the connection is created. 

NTC